### PR TITLE
Update Organisation with new layout and functionality

### PR DESCRIPTION
**Pull request by hoseinmirian**
new organizations layout:

- Smooth link hover effect
- showing grants and type of organization

for testing a record with multiple organizations please use 👍 
http://localhost:8080/#/FAIRsharing.6k0kwd

for testing with no organizations use:
http://localhost:8080/#/45

### CODE REVIEW

File: src/components/Records/Record/Organisations.vue [hunk 1]
```diff
@@ -2,126 +2,148 @@
...
+/* TODO: Replace with query from database */
+import organisationRelations from '@/data/organisationRelations.json';
+export default {
+  name: "Organisations",
+  components: {
+    SectionTitle
+  },
+  data(){
+    return {
+      /* TODO: Replace with query from database */
```
Comment (knirirr) [inline src/components/Records/Record/Organisations.vue]: What's happening with this TODO, and the one at line 78?

**Review by knirirr — CHANGES_REQUESTED** [review]
In general I think the layout looks good. However, there's one special case that's ought to be handled differently, which is multiple grants from the same provider, e.g. 

https://deploy-preview-697--fairsharing.netlify.app/#/3144

It would be better to consolidate all grants into the same list.

Screenshot 2021-02-11 at 14 12 59

File: src/components/Records/Record/Organisations.vue (hunk 1 above)
Comment (hoseinmirian) [reply src/components/Records/Record/Organisations.vue]: I don't know these were left from past. I did not touch those parts

Comment (knirirr) [reply src/components/Records/Record/Organisations.vue]: In which case they are probably left behind from old work, and would need to be removed.

**Comment by hoseinmirian** [comment]
image
duplicate organisation is deleted and if there are 2 grants or more for one organisation all are combined into one array and shown as grant(s) for multiple items and grant for single item

File: src/components/Records/Record/Organisations.vue [hunk 2]
```diff
@@ -2,126 +2,169 @@
...
-          </v-tooltip>
-          <p class="ma-0">
-            {{ organisationLink.grant.name }}
-          </p>
-        </div>
-      </v-card>
+                  <strong v-if="grant">
+                    {{ grant.name }}
+                  </strong>
+                  <span v-if="grantIndex!==organisationLink.grants.length-1"> , </span>
```
Comment (knirirr) [inline src/components/Records/Record/Organisations.vue]: It's a very minor thing, but would you mind removing the space before the comma, please, i.e.:

```
<span v-if="grantIndex!==organisationLink.grants.length-1">, </span>
```

**Review by knirirr — APPROVED** [review]

File: src/components/Records/Record/Organisations.vue (hunk 2 above)
Comment (hoseinmirian) [reply src/components/Records/Record/Organisations.vue]: okay it will be done in next commits

**Pull request merged**